CHANGES.md

```diff
@@ -1,5 +1,7 @@
 # git master
 
+* [486](https://github.com/Eyescale/CMake/pull/486):
+  Fix coverage report generation for non-subprojects
```

Comment:
This is from Daniel's commit but it is correct. It was already working for the subproject case (i.e. a project which is a subproject of viz/stable for instance). Now it is fixed (with a small limitation) for projects which are not a subproject but are their own superproject (thus non-subprojects). Nomenclature is a bit tricky for sure :-)